src/docs/howto-payments/customize-pricing/index.md

```diff
@@ -14,7 +14,7 @@ _**NOTE! In case your pricing relies on the deprecated
 [set-line-items-and-total](/references/transaction-process-actions/#actionset-line-items-and-total)
 action, the old pricing customization article related to that can be
 found
-[here](https://5ee94c280d38f10008a3bfa1--sharetribe-flex-docs-site.netlify.app/docs/cookbook-payments/customize-pricing/).**_
+[here](https://5ee94c280d38f10008a3bfa1--sharetribe-flex-docs-site.netlify.app/docs/background/custom-pricing/).**_
```

Comment:
This is a historical link => the change should be reverted so that the link points to the correct article